tests/cases/unstable/nondet/web/get_webpage_ram_limit.py

```
@@ -0,0 +1,22 @@
+# { "Depends": "py-genlayer:test" }
+import sys
+
+import genlayer as gl
+from genlayer.types import *
```

Ruff (0.15.14)

[error] `from genlayer.types import *` used; unable to detect undefined names

(F403)